Node/package.json

```diff
@@ -11,12 +11,14 @@
   "typings": "./lib/botbuilder-teams.d.ts",
   "dependencies": {
     "botbuilder": "^3.8.0-beta6",
-    "ms-rest": "^1.15.7"
+    "ms-rest": "^1.15.7",
+    "sprintf-js": "^1.0.3"
   },
   "devDependencies": {
     "@types/node": "^7.0.8",
     "mocha": "^2.4.5",
     "restify": "^4.3.0",
-    "typescript": "^2.2.1"
+    "typescript": "^2.2.1",
```

Comment:
@BillZengMSFT We're following the similar implementation to BotBuilder. All card builder class will use `fmtText` function for display text where sprintf-js is required there. You may also check out the original implementation in [Message.ts](https://github.com/Microsoft/BotBuilder/blob/master/Node/core/src/Message.ts) and the usage example in [ThumbnailCard.ts](https://github.com/Microsoft/BotBuilder/blob/master/Node/core/src/cards/ThumbnailCard.ts)